<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>one</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>two</string>
      </property>
     </row>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string>The</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>sky</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>is</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>blue</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_list_items">
     <property name="text">
      <string>List Items</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_item">
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
